### ISSUE TITLE
MyCBE button

### ISSUE THREAD

**Issue by gjwehnes** [Aberhart-school/abe-site#174]
The powers-that-be would like to have a link to a MyCBE button added to the front page.

Potential location is on the RHS above the CBE Open Houses button.

This also does involve the Concussion Protocol links. These are also considered to be not visible enough. Originally, we included them in the same location, but we ended up placing across the bottom.

We do not actually have specific guidelines in the Wiki for adding buttons for external links (outside of the content of the RHS). Thoughts on what those guidelines should be?

![mycbe](https://cloud.githubusercontent.com/assets/6045608/14290692/510b38d4-fb1d-11e5-9b1e-e2472a515982.png)

**Comment by thirstyice** [comment]
Can we not just add it as a link in external links?

**Comment by kid47** [comment]
Or could we just throw it in with the imported content we had from the cbe before if we want to keep the format of the button?


Sent from my (not i)Phone






On Tue, Apr 5, 2016 at 10:18 AM -0700, "thirstyice" <notifications@github.com> wrote:





Can we not just add it as a link in external links?

---

**Comment by gjwehnes** [comment]
I agree that the My CBE button is a fairly straight forward add. Importing the content would be best.

This relates to issue #171, which maybe should be revisited. I did not at the time fully appreciate why the links were moved (correctly) to the center. These really are external links (URL is in comments), and should perhaps be moved back to the RHS.
